### ISSUE TITLE
[Deliverable] Setup Waku Community on dedicated shard with pre-shared key dos protection

### ISSUE THREAD

**Issue by chair28980** [waku-org/pm#262]
Create a Waku token-gated community on its own dedicated shard to further dog food sharding, find and fix any Waku related bugs as well as providing support to the Status app team.

The outputs of this deliverable are:
- Waku token-gated community created on dedicated shard. While this community may not be advertised publicly at first, it will be set up as if, with Waku CC and IFT CCs roles being defined etc.
- Document that summarises the steps to setup a dedicated community, where IFT devops are provisioning the new shard, and setting pre-shared key in coordination with community owner.

**Comment by fryorcraken** [comment]
Agreed with Status team that dedicated shard is a not sustainable way to proceed with scalability for Communities.
